src/common/check_dataset_schema/config.vsh.yaml

```
@@ -0,0 +1,52 @@
+functionality:
+  name: check_dataset_schema
+  namespace: common
+  description: Checks if the dataset has the necessary slots that are predefined in a schema.
+  argument_groups:
+    - name:  Inputs
+      arguments:
+        - name: --input
+          type: file
+          required: true
+          description: A h5ad file.
+        - name: --schema
+          type: file
+          required: true
+          description: A schema file for the h5ad object.
+    - name: Arguments
+      arguments:
+        - name: --stop_on_error
+          type: boolean
+          default: false
+          description: Whether or not to stop with exit code 1 if the input file does not adhere to the schema.
+    - name: Output
+      arguments:
+        - name: --checks
+          type: file
+          required: false
+          description: If specified, this file will contain a structured log of which checks succeeded (or not).
+          example: checks.json
+          direction: output
+        - name: --output
+          type: file
+          required: false
+          description: If specified, the output file will be a copy of the input file.
+          example: output.h5ad
+          direction: output
+  resources:
+    - type: python_script
+      path: script.py
+  test_resources:
+    - path: ../../../resources_test
+    - path: ../../../src
+      dest: src 
```

Is `src/` still needed? We can be a little more specific on the test resources, too:

```suggestion
    - path: ../../../resources_test/common/pancreas
```